client/src/locales/en-US.json

```
@@ -316,9 +316,9 @@
             "sortBySize": "Size",
             "sortByLastUpdated": "Last updated"
         },
-        "getLinkError": "Could not get a link to the workspace",
+        "getLinkError": "Failed to get a link to the workspace",
         "linkCopiedToClipboard": "The link has been copied to the clipboard.",
-        "linkNotCopiedToClipboard": "Could not copy the link. Check that your browser or device allows copy/paste."
+        "linkNotCopiedToClipboard": "Failed to copy the link. Check that your browser or device allows copy/paste."
```

It may not be easy for the user to know how to "check" this. Maybe it is better just to inform it: 
```suggestion
        "linkNotCopiedToClipboard": "Failed to copy link. Your browser or device does not seem to support copy/paste."
```